docs/API-EXAMPLE.md

```diff
@@ -9,12 +9,16 @@ This guide is intended to introduce you to JerryScript embedding API through cre
 #include "jerry.h"
 
 int
-main (int argc, char * argv[]) {
-  char script [] = "print ('Hello, World!');";
+main (int argc, char * argv[])
+{
+  const jerry_api_char_t script[] = "print ('Hello, World!');";
+  size_t script_size = strlen ((const char *) script);
 
-  jerry_completion_code_t code = jerry_run_simple (script,
-                                                   strlen (script),
-                                                   JERRY_FLAG_EMPTY);
+  jerry_completion_code_t return_code = jerry_run_simple (script,
+                                                          script_size,
+                                                          JERRY_FLAG_EMPTY);
+
+  return return_code;
```

Comment:
`jerry_completion_code_t` is an enum. It would be cleaner if `return_code` would be cast to `int`.